### ISSUE TITLE
Running time entry card doesn't update

### ISSUE THREAD

**Issue by pe0ny** [toggl/mobileapp#1195]
1) Start a new time entry or continue existing one
2) Running TE card doesn't appear. *Restart the app*
3) Running TE card appears.
4) Stop running TE.
5) Running TE card doesn't disappear but entry appears in the log list. *Restart the app*
6) Running TE card is hidden now.

- This is reproducible in the latest TF build 1.1 (14), iPhone SE 11.2
- When repeating step 4 (stopping the running entry twice, then app crashes)
- All the changes (starting and stopping) are synced and these appear in webapp, so I believe it's a UI issue.

**Comment by simonrozsival** [comment]
I noticed that when I was debugging the freezing bug - I suspect the subscriptions in the `MainViewModel` are lost - it happened to me when I’ve hidden the app in the background for a while. I discussed this with @willsb and he claimed that this could only happen in the simulator, but apparently, it happens on devices as well.

The problem here is that the UI doesn’t update and reflect the state of the database - taping the stop button crashes the app because the TE isn’t running anymore, but the UI tries to stop it anyway, and you get Realm throwing an exception and crashing the app.
